README.md

```diff
@@ -46,8 +46,8 @@ Kubernetes Fury Monitoring provides the following packages:
 
```

Comment:
Let's not change che main README while we are working on all the packages, we will change this with the last PR